docs/introduction.rst

```diff
@@ -86,10 +86,10 @@ And we plan to do the same for Bitcoin, ZCash, and various other deterministic a
 
 Another example of a cryptocurrency application built on Tendermint is `the Cosmos network <http://cosmos.network>`__.
 
-Fabric, Burrow
-~~~~~~~~~~~~~~
+Other Blockchain Projects
```

Comment:
Do we really need to list "Competition (Collaborators?)"?

Reply:
i think they should be listed here but the section that links to them removed. it's ported from the website; wasn't sure how to to put it

Reply:
Fair, let's remove them